x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_style_property.tsx

```diff
@@ -499,3 +492,21 @@ export function getNumericalMbFeatureStateValue(value: RawValue) {
   const valueAsFloat = parseFloat(value);
   return isNaN(valueAsFloat) ? null : valueAsFloat;
 }
+
+interface PercentilesValues {
+  values?: { [key: string]: number };
+}
+export function percentilesValuesToFieldMeta(
+  percentiles?: PercentilesValues | undefined
+): PercentilesFieldMeta | null {
+  if (percentiles === undefined || percentiles.values === undefined) {
+    return null;
+  }
+  const percentilesFieldMeta = Object.keys(percentiles.values).map((key) => {
+    return {
+      percentile: key,
+      value: percentiles.values![key],
+    };
+  });
+  return _.uniqBy(percentilesFieldMeta, 'value');
```

Comment:
this is guaranteed to take the first (ie lowest) one?

Reply:
Takes the first one.